### DISCUSSION TITLE
Is multiuser case handled by taskwiki?

### DISCUSSION THREAD

**Discussion by jahagirdar** [tbabej/taskwiki#333] [Q&A] [answered]
What happens if 

1. I create a taskwiki header,
2. populate it with a checklist,
3. Save it to a shared folder.
4. share the file with another taskwiki user.
5. The user opens the file in a taskwiki enhanced vim

Will taskwiki create the project, deadlines, tags and the tasks for the second user? If yes, can this method be used by two people to synchronize their tasklist? Are there any known corner cases

**Comment by tbabej** [comment, marked as answer]
This use case is not handled natively. In order for this to work properly, the two people would need to use `task sync` to synchronize their tasklists (which syncs _all_ the tasks and modifications).

Alternatively, you can share a task list in a taskwiki format and have the receiving user delete all the short UUID comment identifiers (the `  #af739cb` at the end of each task), however, that will only help him generate the same initial task list, it won't cope with eventual updates.
